lib/registries/addon/my-registrations/template.hbs

```diff
@@ -1,6 +1,8 @@
-{{title (t 'registries.my_registrations.header')}}
+{{page-title (t 'registries.my_registrations.header')}}
```

Comment:
Does this refer to the node package install from here: https://www.npmjs.com/package/ember-page-title?

Reply:
It looks like it. I assumed it was a native ember thing, but it looks like they pre-installed in new ember apps https://guides.emberjs.com/release/accessibility/page-template-considerations/

Reply:
Ok, got it. I am unfamiliar with the dependency and was unsure how it would affect scalability. Judging from the documentation, this is very helpful with accessibility issues including the pushState() method we may want to look into. 

Product mentioned accessibility is a core focus for our users but I also see this applicable towards in-situ changes within the OSF as well.